src/components/AddComment.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { useContext } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import { addComment } from "../apis/comments";
+import { UserContext } from "../context/UserContext";
+import { Button } from "react-bootstrap";
+
+const AddComment = () => {
+  const [comment, setComment] = useState("");
+  const { review_id } = useParams();
+  const { user } = useContext(UserContext);
+  const navigate = useNavigate();
+  const successMessage = document.getElementById("message");
+
+  const createUserComment = async () => {
+    try {
+      const commentObj = { username: user.username, body: comment };
+      await addComment(review_id, commentObj);
+      successMessage.innerText = `Thank you ${user.name} for your comment`;
+      successMessage.style.color = "blue";
+      setComment("");
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (comment === "") {
+      successMessage.innerText = `You need to write you comment before you can submit it`;
+      successMessage.style.color = "red";
+    } else {
+      createUserComment();
+    }
+  };
+
+  const handelTextChange = (e) => {
+    setComment(e.target.value);
+    successMessage.innerText = "";
```

Comment:
This line is giving me an error when I try to type anything. Cannot set properties of null (setting 'innerText'). Also do need to be using DOM manipulation? Try your best to use React exclusively and not combining the two